test/units/testsuite-80-fails-on-restart.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+exit 1
```

Comment:
You can just inline this in the service unit: `sh -c "exit 1"`

Reply:
> You can just inline this in the service unit: `sh -c "exit 1"`

... or use `false` instead.